source/news/2017-05-15.html.markdown

```diff
@@ -0,0 +1,77 @@
+---
+title: dry-view 0.3.0: another step towards solving the view layer mess
+date: 2017-05-15
+author: timriley
+---
+
+# dry-view 0.3.0: another step towards solving the view layer mess
+
+We’re excited to announce the release of dry-view 0.3.0, which introduces two concepts for better organising your views: view parts and decorators.
+
+How do these work? Every dry-view controller now comes configured with a **decorator** (you can use our default or provide your own), whose job it is to wrap up your values in **view parts** before passing them to the template. View parts give you a place to cleanly combine view-specific behaviour with your application’s values.
+
+By default, every view part is a plain `Dry::View::Part` instance. This gives you access to a `#render` method for rendering a partial with the view part included in the partial’s scope:
+
+```ruby
+class AccountView < Dry::View::Controller
+  configure do |config|
+    config.template = "account"
+  end
+
+  expose :account do
+    # fetch the account value
+  end
+end
+```
+
+```erb
+<h1>Your account</h1>
+
+<%# Renders "account/_info_box.html.erb" with `account` in scope %>
+<%= account.render :info_box %>
+```
+
+This is just the beginning. Things get interesting when you specify your own view part classes:
+
+```ruby
+expose :account, as: AccountPart
+```
+
+Since view parts are designed as wrappers, you have access to every method on the value, made easily available via `method_missing` behaviour:
+
```

Comment:
Not sure if it's worth mentioning `method_missing`, some folks are allergic to its usage.